index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous">
+
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+
+	<script src="https://kit.fontawesome.com/7aab0fc5d4.js" crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="style.css">
+	<title>My Capstone Project</title>
+</head>
+<body>
+	<header class="page-heading">
+		<div class="nav-top">
+			<ul class="social__list">
+				<li class="social__item">
+					<a href="https://www.facebook.com/ReggaeSumfest" class="social__link">
+						<i class="fa-brands fa-facebook-f"></i></a>
+					</li>
+				<li class="social__item">
+					<a href="https://twitter.com" class="social__link">
+						<i class="fa-brands fa-twitter"></i></a>
+					</li>
+				<li class="social__item">
+					<a href="#" class="social__link">English</a>
+				</li>
+				<li class="social__item">
+					<a href="#" class="social__link">My Page</a>
+				</li>
+				<li class="social__item">
+					<a href="#" class="social__link">Logout</a></li>
+			</ul>
+		</div>
+		<button id="menu-close" class="nav-toggle" type="button" aria-label="toggle navigation">
+				<i class="fa-solid fa-bars"></i>
+		</button>
+		<nav class="nav">
+			<div class="nav__logo">
+				<a href="#"><img class="logo" src="./assets/logos/logo-nav.png" alt="logo"></a>
```

Comment:
- Hey, please, kindly make the logo link to the homepage by making it a live link to the index.html page and not a dead link.